src/landingPage/LandingPage.module.scss

```diff
@@ -7,17 +7,7 @@
   --light-background: #e5e7eb;
   --bright-text: #e3e3e4;
   --fade-text: #767a82;
-  --std-gradient: linear-gradient(270deg, #2af588 0%, #334675 99.99%);
+  --std-gradient: linear-gradient(90deg, #2AF588 0%, #334675 100%);
```

Comment:
Why are we changing the angle to 90deg?

Reply:
Because somehow, even though "copy as css" from figma says 270deg, 90drg actually matches what the design in figma shows.